### DISCUSSION TITLE
Import Export vcpkg

### DISCUSSION THREAD

**Discussion by jean-christophe81** [microsoft/vcpkg#39947] [General]
Libraries compilations of my project take a long time (about one hour) on windows runner. So I run vcpkg install on a workflow and I want to use this compilation result on my project compilation workflow. 
I have started to use vcpkg export --7zip, and I follow instructions to reuse it in my second workflow, but I didn't manage to use it, vcpkg always complained about missing things.
So I have decided to zip entire vcpkg directory and dezip it in my second workflow.

Have you a better solution?

Regards

Jean Christophe

**Comment by AugP** [comment]
If compilation time of your dependencies is the core issue, have you taken a look at binary caching? https://learn.microsoft.com/vcpkg/consume/binary-caching-overview
https://learn.microsoft.com/vcpkg/users/binarycaching
